lib/twitter_cldr/js/mustache/tokenizers/numbers/number_tokenizer.coffee

```diff
@@ -0,0 +1,46 @@
+# Copyright 2012 Twitter, Inc
+# http://www.apache.org/licenses/LICENSE-2.0
+
+class TwitterCldr.NumberTokenizer
+  constructor : (@data_reader) ->
+    @special_symbols_map = {
+      '.' : '{DOT}',
+      ',' : '{COMMA}',
+      '0' : '{ZERO}',
+      '#' : '{POUND}',
+      '¤' : '{CURRENCY}',
+      '%' : '{PERCENT}',
+      'E' : '{SCIENTIFIC}'
+    }
+
+    @inverse_special_symbols_map = {}
+    for k, v of @special_symbols_map
+      @inverse_special_symbols_map[v] = k
+
+    @special_symbols_regex = new RegExp("'(?:" + [TwitterCldr.Utilities.regex_escape(k) for k, v of @special_symbols_map].join('|') + ")'")
```

Comment:
Changed to: 

```coffeescript

@special_symbols_regex = new RegExp("'(?:" + (
    [(TwitterCldr.Utilities.regex_escape(k)) for k, v of @special_symbols_map]
  ).join('|') + ")'")
@inverse_special_symbols_regex = new RegExp(
  (
    [TwitterCldr.Utilities.regex_escape(k) for k, v of @inverse_special_symbols_map]
  ).join('|'))
```

It doesn't help much, though. These list comprehensions are a bit tricky to shorten/split.